### PR TITLE
uint -> uint256

### PR DESCRIPTION
**Pull request by axchu**
### Fixes
- Fixes # https://github.com/ethereum-optimism/roadmap/issues/284

## Contributing Agreement

- [x] I have read and understood the [Optimism Contributing Guide and Code of Conduct](https://github.com/ethereum-optimism/optimism-monorepo/blob/master/.github/CONTRIBUTING.md) and am following those guidelines in this pull request.

### CODE REVIEW

**Review by ben-chain — APPROVED** [review]
LGTM!  Let's hold off on merging this one just yet, need to confirm that changing the `L2ToL1` event hash is not gonna break some other things.

File: contracts/optimistic-ethereum/iOVM/precompiles/iOVM_L2ToL1MessagePasser.sol [hunk 1]
```diff
@@ -11,7 +11,7 @@ interface iOVM_L2ToL1MessagePasser {
      **********/
 
     event L2ToL1Message(
-       uint _nonce,
+       uint256 _nonce,
```
Comment (kfichter) [inline contracts/optimistic-ethereum/iOVM/precompiles/iOVM_L2ToL1MessagePasser.sol:14]: Hmmm the indent seems wrong here, not sure how that happened.

**Comment by kfichter** [comment]
> LGTM! Let's hold off on merging this one just yet, need to confirm that changing the `L2ToL1` event hash is not gonna break some other things.

Just tested in remix, event hash doesn't change when switching from `uint` to `uint256`.

File: contracts/optimistic-ethereum/iOVM/precompiles/iOVM_L2ToL1MessagePasser.sol (hunk 1 above)
Comment (axchu) [reply contracts/optimistic-ethereum/iOVM/precompiles/iOVM_L2ToL1MessagePasser.sol]: fixed!

**Comment by ben-chain** [comment]
This looks good to go, based on what @kfichter found, should be good to merge to master.

**Pull request merged**